### PR TITLE
Springify util classes that have dependencies; Allow reuse of data pipeline schedule

### PR DESCRIPTION
**Pull request by liujoshua**

### CODE REVIEW

File: src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java [hunk 1]
```diff
@@ -16,27 +17,40 @@
  * Creates a Data Pipeline for backing up Dynamo DB tables within a region.
  */
 public class DynamoDataPipelineHelper {
+    public static final class PipelineObjectType {
+        public static final String DYNAMO_DATA_NODE = "DynamoDBDataNode";
+        public static final String S3_DATA_NODE = "S3DataNode";
+        public static final String EMR_ACTIVITY = "EmrActivity";
+        public static final String SCHEDULE = "Schedule";
+        public static final String EMR_CLUSTER = "EmrCluster";
+    }
```
Comment (alxdarksage) [inline src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java:26]: Any value to this being an enum?

File: src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoNamingHelper.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapperConfig;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import org.sagebionetworks.bridge.config.Config;
+import org.sagebionetworks.bridge.config.Environment;
+
+/**
+ * Created by liujoshua on 4/18/16.
+ */
+public class DynamoNamingHelper {
```
Comment (alxdarksage) [inline src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoNamingHelper.java:13]: Is this a Spring bean? How do we inject Config in the constructor?

File: src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoNamingHelperTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        config = mock(Config.class);
+        namingHelper = new DynamoNamingHelper(config);
+
+        when(config.getUser()).thenReturn(USER);
+        when(config.getEnvironment()).thenReturn(ENV);
+    }
+
+    @Test
+    public void testGetSimpleTableName() {
+        assertEquals("TableName", namingHelper.getSimpleTableName(EXPECTED_PREFIX + "TableName"));
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoNamingHelperTest.java:36]: Can we screw this up by putting ".TableName" or "tableName" or no expected prefix... ?

File: src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoNamingHelper.java (hunk 2 above)
Comment (liujoshua) [inline src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoNamingHelper.java:13]: I'll declare it a bean in BridgeSpringConfig (and handle Config there)

I don't think we make anything a bean in bridge-base

File: src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java (hunk 1 above)
Comment (liujoshua) [inline src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java:26]: I don't think there will be an advantage

**Pull request merged**